app/controllers/v1/artists_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::ArtistsController < ApplicationController
   def index
-    artists = Artist.all
+    artists = Artist.all.includes(:songs)
```

Comment:
You actually don't need to call `.all` here - it should return all your artists just calling `includes`